### ISSUE TITLE
Using OpenApePose for multi-Ape dataset

### ISSUE THREAD

**Issue by yanavdsande** [desai-nisarg/OpenApePose#1]
Hi there, 

First of all, thank you for your work. I found it very valuable and I much appreciate that you published the dataset, model and annotation files. 

I am currently working on a project about grooming. I have a dataset with video material of chimpanzees grooming each other in the wild. My data depicts multiple apes at the same time. This means that when I convert my annotations to your annotation format I run into the problem of having the annotations of multiple animals. Would it be possible to use your dataset and model on my multi individual dataset? 

Specifically, I was thinking that I could use your model to begin with and fine tune using my dataset (however, if I understand it correctly, my model input must be in the same format as your model input, and that is where the problem of multiple individuals come in). Is there a clever way to work around this or would it be an idea to include an extra label in your annotation files and train again? Or can I use your dataset to pre-train for instance DLC multi animal and finetune on my own dataset? 

In the best case scenario, I would like to compare DLC multi animal with mmpose and also control the effect of larger datasets/pretraining on an ape dataset.

I am curious to hear your thoughts! 

Thank you in advance!

**Comment by desai-nisarg** [comment]
Hi,

The model predicts keypoints in the space of the image specified by a bounding box, and it's fine to have multiple bounding boxes in the same image. One way to make it work is to match our annotation format such that you have multiple annotations with the same filename but different bounding boxes and key points.

Hopefully that helps, but let me know if that doesn't work.

**Comment by yanavdsande** [comment]
Thanks! I will give it a try and let you know! :) I also wrote a program to reformat openapepose json annotations to DLC format annotations. I will link that too, perhaps it is of use for anyone.

**Comment by desai-nisarg** [comment]
That sounds great! I'd be happy to merge if you wanted to set up a PR!
